google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -64,15 +69,27 @@ public String getDisplayString() {
           public OSProcessHandler createProcessHandler(
               String workingDirectory, Map<String, String> envVariables) throws ExecutionException {
             CloudSdkService sdkService = CloudSdkService.getInstance();
+
             if (sdkService.getSdkHomePath() == null
-                || sdkService.getSdkHomePath().toString().isEmpty()) {
+                || StringUtils.isEmpty(sdkService.getSdkHomePath().toString())) {
```

Comment:
In that case, I'd argue the code would be a bit cleaner to revert back to String.isEmpty() (and we'd cut the dependency on that apache lib).

But, ideally, what I'd really like is to get rid of this empty check altogether, for the reasons on the above comment.